javascript/core/intro-js/javascript-basics-part-1.md

```diff
@@ -39,7 +39,7 @@ Like most languages, there are a number of reserved words in JavaScript that you
 
 ### Identifier naming rules
 
-When naming any function, property or variable, the first character must be a letter, $ sign or underscore.
+When naming any function, property or variable, the first character mustn't be a letter, $ sign or underscore.
```

Comment:
Hi @Kriof, thanks for your contribution!

Unfortunately, your suggestion is incorrect :)

See [this discussion](https://github.com/enkidevs/curriculum/pull/1288#discussion_r272889001) for more details